app/api/test-email/route.ts

```diff
@@ -1,19 +1,36 @@
 import { NextResponse } from "next/server";
 import { LambdaClient, InvokeCommand } from "@aws-sdk/client-lambda";
 import { z } from "zod";
+import { execSync } from "child_process";
 import { LAMBDA_FUNCTION_NAMES } from "../../lib/constants";
 
 const testRequestSchema = z.object({
-  previewBranchName: z.string().min(1),
-  testSubject: z.string().optional(),
-  testSender: z.string().email().optional()
+  testSubject: z.string().min(1),
+  testSender: z.string().email().min(1)
 });
 
+function getCurrentBranch(): string {
+  try {
+    const branchName = execSync('git branch --show-current', {
+      encoding: 'utf8'
+    }).trim();
```

Comment:
We cannot use `git` in the vercel edge function. Get the branch name either dynamically using the GitHub api or with an environment variable, whichever's easier